### ISSUE TITLE
Crash when using Item Browser

### ISSUE THREAD

**Issue by kurisubrooks** [medeor413/starcheat#17]
``` python
Traceback (most recent call last):
  File "build\gui\itembrowser.py", line 159, in update_item_view
  File "build\assets\items.py", line 141, in get_item_icon
AttributeError: 'list' object has no attribute 'split'
```

``` python
Traceback (most recent call last):
  File "build\gui\itembrowser.py", line 159, in update_item_view
  File "build\assets\items.py", line 137, in get_item_icon
TypeError: argument of type 'NoneType' is not iterable
```

**Comment by kurisubrooks** [comment]
It seems to happen when selecting the 12th item in the dropdown

**Comment by rubellyte** [comment]
Fixed by a1b048de21496aba0b99d5f3e74bc67e3b766f8f
